asn1ate/sema.py

```diff
@@ -719,6 +721,18 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self,elements):
+	self.module2symbols = { }
```

Comment:
I should read up on the `IMPORTS` and `EXPORTS` syntax, but can you describe what this does?

What is the semantics of the mapping here?